arm-recoveryservices/2016-06-01/swagger/registeredidentities.json

```diff
@@ -0,0 +1,422 @@
+            "in": "path",
+            "description": "Certificate friendly name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "certificateRequest",
+            "in": "body",
+            "description": "Input parameters for uploading the vault certificate.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CertificateRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VaultCertificateResponse"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/registeredIdentities/{identityName}": {
+      "delete": {
+        "tags": [
+          "RegisteredIdentities"
+        ],
+        "description": "Unregisters the given container from your Recovery Services vault.",
+        "operationId": "RegisteredIdentities_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "name": "identityName",
+            "in": "path",
+            "description": "Name of the protection container to unregister.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "NoContent"
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "CertificateRequest": {
+      "description": "Details of the certificate to be uploaded to the vault.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/RawCertificateData"
+        }
+      }
+    },
+    "RawCertificateData": {
+      "description": "Raw certificate data.",
+      "type": "object",
+      "properties": {
+        "authType": {
+          "description": "Specifies the authentication type.",
+          "enum": [
+            "Invalid",
+            "ACS",
+            "AAD",
+            "AccessControlService",
+            "AzureActiveDirectory"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AuthType",
+            "modelAsString": true
+          }
+        },
+        "certificate": {
+          "format": "byte",
+          "description": "The base64 encoded certificate raw data string",
+          "type": "string"
+        }
+      }
+    },
+    "ResourceCertificateAndAADDetails": {

```

Comment:
Unless I'm mistaken there are duplicate properties in ```ResourceCertificateAndAADDetails``` and ```ResourceCertificateAndACSDetails```, why not move them in ```ResourceCertificateDetails```